provisioning/roles/wazuh/ansible-filebeat-oss/defaults/main.yml

```diff
@@ -9,7 +9,7 @@ filebeat_output_indexer_hosts:
   - "localhost:9200"
 
 filebeat_module_package_url: https://packages.wazuh.com/4.x/filebeat
-filebeat_module_package_name: wazuh-filebeat-0.3.tar.gz
+filebeat_module_package_name: wazuh-filebeat-0.2.tar.gz
```

Comment:
```suggestion
filebeat_module_package_name: wazuh-filebeat-0.3.tar.gz
```

Reply:
Filebeat 3 is not in production. Currently with this change, instances will be not provisioned correctly.

Reply:
Filebeat is now in production

Reply:
This will be done in  https://github.com/wazuh/wazuh-qa/issues/4591